specs/ceph_crud_flows.adoc

```diff
@@ -0,0 +1,557 @@
+          help: rule set failure domain for ec profile
+          type: String
+      enabled: true
+      list: clusters/$TendrlContext.integration_id/ec_profiles
+      value: clusters/$TendrlContext.integration_id/ec_profiles/$ECProfile.name
+      help: EC profile
+    Pool:
+      atoms:
+        Create:
+          enabled: true
+          help: "Pool create Atom"
+          inputs:
+            mandatory:
+              - Pool.poolname
+              - Pool.pg_num
+              - Pool.min_size
+            optional:
+              - Pool.type
+              - Pool.erasure_code_profile
+              - Pool.quota_enabled
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          name: "Create Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.create.Create
+          type: Create
+          uuid: bd0155a8-ff15-42ff-9c76-5176f53c13e0
+        Delete:
+          enabled: true
+          help: "Pool delete Atom"
+          inputs:
+            mandatory:
+              - Pool.pool_id
+          name: "Delete Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.delete.Delete
+          type: Delete
+          uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3720
+        Update:
+          enabled: true
+          help: "Pool update Atom"
+          inputs:
+            mandatory:
+              - Pool.pool_id
+            optional:
+              - Pool.poolname
+              - Pool.size
+              - Pool.pg_num
+              - Pool.quota_enabled
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          name: "Update Pool"
+          run: tendrl.ceph_integration.objects.Pool.atoms.update.Update
+          type: Update
+          uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3721
+      flows:
+        DeletePool:
+          atoms:
+            - tendrl.ceph_integration.objects.Pool.atoms.delete.Delete
+          help: "Delete Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+              - TendrlContext.sds_name
+              - TendrlContext.sds_version
+              - TendrlContext.integration_id
+          run: tendrl.ceph_integration.objects.Pool.flows.delete_pool.DeletePool
+          type: Delete
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3b9
+        UpdatePool:
+          atoms:
+            - tendrl.ceph_integration.objects.Pool.atoms.update.Update
+          help: "Update Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+            optional:
+              - Pool.poolname
+              - Pool.size
+              - Pool.pg_num
+              - Pool.quota_enabled
+              - Pool.quota_max_objects
+              - Pool.quota_max_bytes
+          run: tendrl.ceph_integration.objects.Pool.flows.update_pool.UpdatePool
+          type: Update
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3b2
+      attrs:
```

Comment:
I don't see eg. expected-num-object listed there. We may need to check how is this aligned with http://docs.ceph.com/docs/master/rados/operations/pools/